cli/src/pcluster/aws/aws_resources.py

```diff
@@ -137,6 +138,22 @@ def launch_time(self):
         """Return launch time of the instance."""
         return self._instance_data.get("LaunchTime")
 
+    @property
+    def node_type(self) -> str:
+        """Return node type of the instance."""
+        return self._get_tag_value(PCLUSTER_NODE_TYPE_TAG)
+
+    @property
+    def queue_name(self) -> str:
+        """Return queue name of the instance."""
+        return self._get_tag_value("QueueName")
```

Comment:
Can we align this tag to follow same convention of `parallelcluster:node-type` -> `parallelcluster:queue-name`?

Reply:
I will align this in the next PR

Reply:
Done with a new commit in this PR